### PR TITLE
Karma: Add custom matcher for aXe accessibility violations

### LINKED ISSUE #4019: Karma: include aXe accessibility testing
## Feature Description

Some automated accessibility testing is already possible today, but it hasn’t been fully configured and documented yet. Also, it’s not yet in place for integrations tests, which would be the ideal place for this.

An example implementation exists at https://github.com/dequelabs/axe-core/tree/develop/doc/examples/jasmine

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Usage: `await expectAsync(node).toHaveNoViolations();`

An optional [`options`](https://www.deque.com/axe/core-documentation/api-documentation/#options-parameter) object can be passed to `toHaveNoViolations()` to configure aXe.

**Example**

I added `await expectAsync(layerPanel.node).toHaveNoViolations();` to `assets/src/edit-story/components/panels/layer/karma/layer.karma.js` and got the following result:

```
     Expected element to pass aXe accessibility tests. Violations found:
            [ color-contrast] Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds
Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=axeAPI
Impact: serious


HTML: <div class="layer__BackgroundDescription-sc-18ebmzf-4 gSyHal">Background (locked)</div>
Fix any of the following:
  Element has insufficient color contrast of 4.22 (foreground color: #9b9c9c, background color: #373939, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

```

## To-do

* [ ] Set up some default options for aXe
* [ ] Documentation
* [ ] Use in tests

## Testing Instructions

1. Add `await expectAsync(node).toHaveNoViolations();` assertion to a test
1. Run tests
1. Verify results

---

Fixes #4019

### CODE REVIEW

File: karma-dashboard.config.cjs [hunk 1]
```diff
@@ -116,7 +118,7 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
 
     // enable / disable watching file and executing tests whenever any file changes
-    autoWatch: true,
+    autoWatch: false,
```
Comment (dvoytenko) [inline karma-dashboard.config.cjs]: Is this necessary?

File: assets/src/edit-story/components/canvas/karma/cloneSelection.karma.js [hunk 2]
```diff
@@ -212,49 +211,3 @@ describe('Clone element integration', () => {
     );
   }
 });
-
-const customMatchers = {
```
Comment (dvoytenko) [inline assets/src/edit-story/components/canvas/karma/cloneSelection.karma.js:216]: Are these just not used?

Comment (swissspidy) [reply assets/src/edit-story/components/canvas/karma/cloneSelection.karma.js:216]: They are used, but already defined in `karma/fixture/init.js`. So this is just duplicated code.

File: karma-dashboard.config.cjs (hunk 1 above)
Comment (swissspidy) [reply karma-dashboard.config.cjs]: A leftover from testing, can undo.

Although worth noting that re-running the whole test suite whenever any file changes is expensive.

Comment (dvoytenko) [reply karma-dashboard.config.cjs]: I normally would set `fit` or `fdescribe` on one test and keep rerunning until it works. Is this what controls that?

Comment (swissspidy) [reply karma-dashboard.config.cjs]: Yep that's the one.

I will undo the change then to not break that workflow.

**Comment by swissspidy** [comment]
@carlos-kelly @barklund Adding you as reviewers to hear your thoughts on the suggested API.

**Review by dvoytenko — APPROVED** [review]

**Pull request merged**